api/envoy/extensions/filters/http/ext_authz/v3/ext_authz.proto

```diff
@@ -292,10 +292,30 @@ message ExtAuthz {
   // If unset, defaults to true.
   google.protobuf.BoolValue enable_dynamic_metadata_ingestion = 27;
 
+  // Deprecated in favor of
+  // :ref:`LoggingOptions.filter_metadata<envoy_v3_api_field_extensions.filters.http.ext_authz.v3.LoggingOptions.filter_metadata>`.
+  google.protobuf.Struct filter_metadata = 28
```

Comment:
For existing users of this field that are not getting stats emitted, will that behavior continue, or will they suddenly start getting the stats emitted?  If the latter, is that going to break anything for anyone?

Reply:
Existing users of this field, if they exist, won't get their filter metadata anymore. We think it's unlikely anyone will use this field without stats.

Reply:
This is technically a breaking change, which we shouldn't allow.  How do we know there aren't existing users of this field?

Reply:
Point taken, working on addressing it. Thanks